libqtile/core/manager.py

```diff
@@ -288,9 +286,15 @@ def _process_screens(self) -> None:
             if not self.current_screen:
                 self.current_screen = scr
 
-            scr._configure(
-                self, i, x, y, w, h, self.groups[i],
-            )
+            try:
```

Comment:
mind if this is an `if len(self.groups) > i:` instead of a `try ... except`? exceptions in python are relatively expensive compared to a branch.

Reply:
Good point, cheers